Error: Cannot find module '@/spotify'

Something is iffy with the alias, @, as it compilles through rsc but still gives these issues if used by `yarn dev`.

Ever figured out that problem? Getting similar results.

`ubuntu@ubuntu:~/twitch-spotify-request-bot$ yarn start
yarn run v1.22.17
$ node dist/index.js
internal/modules/cjs/loader.js:905
  throw err;
  ^

Error: Cannot find module '@/spotify'
Require stack:
- /home/ubuntu/twitch-spotify-request-bot/dist/index.js
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:902:15)
    at Function.Module._load (internal/modules/cjs/loader.js:746:27)
    at Module.require (internal/modules/cjs/loader.js:974:19)
    at require (internal/modules/cjs/helpers.js:101:18)
    at Object.<anonymous> (/home/ubuntu/twitch-spotify-request-bot/dist/index.js:32:19)
    at Module._compile (internal/modules/cjs/loader.js:1085:14)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1114:10)
    at Module.load (internal/modules/cjs/loader.js:950:32)
    at Function.Module._load (internal/modules/cjs/loader.js:790:12)
    at Function.executeUserEntryPoint [as runMain] (internal/modules/run_main.js:75:12) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [ '/home/ubuntu/twitch-spotify-request-bot/dist/index.js' ]
}
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.`

--- a/src/auth/auth-server.ts
+++ b/src/auth/auth-server.ts
@@ -1,8 +1,8 @@
 import env from 'env-smart';
 import express from 'express';
 
-import { SpotifyService } from '@/spotify';
-import { envDirectory } from '@/utils';
+import { SpotifyService } from '../spotify';
+import { envDirectory } from '../utils';
 
 import 'dotenv/config';
 

--- a/src/index.ts
+++ b/src/index.ts
@@ -1,7 +1,7 @@
 import * as path from 'path';
 
-import { SpotifyService } from '@/spotify';
-import { TwitchService } from '@/twitch';
+import { SpotifyService } from './spotify';
+import { TwitchService } from './twitch';
 
 // Required for pkg to recognise these files as assets
 path.join(__dirname, '../.env');

--- a/src/spotify/spotify.service.ts
+++ b/src/spotify/spotify.service.ts
@@ -5,8 +5,8 @@ import fs from 'fs';
 import env from 'env-smart';
 import SpotifyWebApi from 'spotify-web-api-node';
 
-import { waitForCode, SpotifyAuth } from '@/auth';
-import { envDirectory } from '@/utils';
+import { SpotifyAuth, waitForCode } from '../auth';
+import { envDirectory } from '../utils';
 
 env.load({ directory: envDirectory });
 

--- a/src/twitch/twitch.service.ts
+++ b/src/twitch/twitch.service.ts
@@ -1,8 +1,8 @@
 import env from 'env-smart';
 import tmi, { ChatUserstate } from 'tmi.js';
 
-import { SpotifyService } from '@/spotify';
-import { envDirectory, getTrackIdFromLink, SPOTIFY_LINK_START } from '@/utils';
+import { SpotifyService } from '../spotify';
+import { envDirectory, getTrackIdFromLink, SPOTIFY_LINK_START } from '../utils';
 
 env.load({ directory: envDirectory });
 

--- a/tsconfig.json
+++ b/tsconfig.json
@@ -10,11 +10,7 @@
     "esModuleInterop": true,
     "resolveJsonModule": true,
     "skipLibCheck": true,
-    "forceConsistentCasingInFileNames": true,
-    "baseUrl": "./src/",
-    "paths": {
-      "@/*": ["./*"]
-    }
+    "forceConsistentCasingInFileNames": true
   },
   "include": ["**/*.ts"],
   "exclude": ["dist", "node_modules"]

